SPECS-EXTENDED/gdisk/gdisk.spec

```diff
@@ -1,18 +1,17 @@
-Summary:        An fdisk-like partitioning tool for GPT disks
-Name:           gdisk
-Version:        1.0.9
-Release:        3%{?dist}
-License:        GPLv2
-Vendor:         Microsoft Corporation
-Distribution:   Azure Linux
-URL:            https://www.rodsbooks.com/gdisk/
-Source0:        https://downloads.sourceforge.net/gptfdisk/gptfdisk-%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Patch0:         gdisk-libuuid.patch
-BuildRequires:  gcc-c++
-BuildRequires:  libuuid-devel
-BuildRequires:  make
-BuildRequires:  ncurses-devel
-BuildRequires:  popt-devel
```

Comment:
Please retain the original order of the tags Summary, Name, Version, Release etc. It would be great if you can add only the differences for the upgrade.